src/tests/ftest/util/ec_utils.py

```diff
@@ -34,6 +34,27 @@ def get_data_parity_number(log, oclass):
     tmp = re.findall(r'\d+', oclass)
     return {'data': tmp[0], 'parity': tmp[1]}
 
+def check_aggregation_status(pool):
+    """EC Aggregation triggered status.
+    Args:
+        pool: pool object to get the query.
```

Comment:
Minor: pool (object)

Reply:
Let me handle this in the next PR instead of rerunning full CI again.